### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/zh/config/giscus.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#495] [Announcements]
# https://plugin-comment2.vuejs.press/zh/config/giscus.html

repo 类型: string; 必填: 是; 存放评论的仓库 repoId 类型: string; 必填: 是; 仓库 ID，请从 Giscus 页面 (https://giscus.app/zh-CN) 生成。 category 类型: string; 必填: 是; 讨论分类 categoryId 类型: string; 必填: 是; 讨论分类 I...

https://plugin-comment2.vuejs.press/zh/config/giscus.html

**Comment by hwbrzzl** [comment]


**Reply by Mister-Hope** [reply]
这跟我有毛关系？

![image](https://user-images.githubusercontent.com/33315834/225545550-bf772ec3-b2ca-4879-a215-3e2883348528.png)

我能到你仓库去加这个文件？

**Comment by WL2O2O** [comment]
hello

**Reply by yinuoguan** [reply]
可以

**Reply by yinuoguan** [reply]
没问题

**Comment by ylnlh** [comment]
为什么我的提示 giscus.app 链接不上

**Comment by zhanglinweigithub** [comment]
您好，请问一下，vuepress-theme-hope主题使用了Giscus评论功能，如何添加浏览量？还是只有Waline才支持显示页面浏览量？

**Reply by Mister-Hope** [reply]
giscus没有浏览量功能
